docs/source/llms/index.rst

```diff
@@ -239,6 +239,15 @@ Harnessing the power of LLMs becomes effortless with flavors designed specifical
   - All essential tools and functionalities consolidated under MLflow.
   - Focus on deriving value from LLMs without getting bogged down by interfacing and optimization intricacies.
 
+.. toctree::
```

Comment:
This is not relevant but to improve side-bar navigation. Previously the tree was located under `h2`, which hide all headings of the LangChain page from side bar due to hitting the max depth. Moving it to `h1` make them visible and easy to navigate.

**Before** ("MLflow LangChain Flavor" is at max depth so sections are not shown)
<img width="300" alt="Screenshot 2024-06-11 at 0 58 50" src="https://github.com/mlflow/mlflow/assets/31463517/4aa96347-6bc0-4485-9c8b-45b4c6ca1393">

**After**
<img width="300" alt="Screenshot 2024-06-11 at 0 59 10" src="https://github.com/mlflow/mlflow/assets/31463517/2ac7e13e-1d8f-4180-aa63-670334da5923">